recipe/meta.yaml

```
@@ -68,7 +68,7 @@ requirements:
     - gflags
     - msgpack-c
     - aws-sdk-cpp
-    - azure-core-cpp
+    - azure-core-cpp >=1.11.0
```

Currently [this overwrites](https://github.com/conda-forge/arcticdb-feedstock/pull/233/files#diff-c399e81833f8217983f67b22ce193230d00ffae438a62252a1538fb602479cb3L7-L8) [those specifications](https://github.com/conda-forge/arcticdb-feedstock/blob/rc/recipe/conda_build_config.yaml#L15-L39).

For some more precision, could this pin be ported [here](https://github.com/conda-forge/arcticdb-feedstock/blob/rc/recipe/conda_build_config.yaml#L15-L39) not to overwrite them with this comment:

```suggestion
    # Pinned to >=1.11.0 for the `main` specification
    # See `./conda_build_config.yaml`.
    - azure-core-cpp
```

?